x-pack/plugins/enterprise_search/public/applications/workplace_search/index.tsx

```diff
@@ -20,21 +21,19 @@ import { SetupGuide } from './views/setup_guide';
 import { ErrorState } from './views/error_state';
 import { Overview } from './views/overview';
 
-export const WorkplaceSearch: React.FC<IInitialAppData> = () => {
+export const WorkplaceSearch: React.FC<IInitialAppData> = (props) => {
   const { config } = useContext(KibanaContext) as IKibanaContext;
+  return !config.host ? <WorkplaceSearchUnconfigured /> : <WorkplaceSearchConfigured {...props} />;
+};
+
+export const WorkplaceSearchConfigured: React.FC<IInitialAppData> = (props) => {
+  const { hasInitialized } = useValues(AppLogic) as IAppValues;
+  const { initializeAppData } = useActions(AppLogic) as IAppActions;
```

Comment:
Reminder to self: I should update App Search's `IAppLogicValues/Actions` types to remove the `Logic`